crates/autopilot/src/arguments.rs

```diff
@@ -59,6 +59,10 @@ pub struct Arguments {
     #[clap(long, env, default_value = "0.0.0.0:9589")]
     pub metrics_address: SocketAddr,
 
+    /// Address to bind the native price API server
+    #[clap(long, env, default_value = "0.0.0.0:12088")]
+    pub native_price_api_address: SocketAddr,
```

Comment:
Probably not necessary to tie all the API stuff to native prices alone. Having an HTTP server in the autopilot can come in handy for other stuff as well. The comment applies to a bunch of naming in this PR.